build.gradle

```diff
@@ -40,6 +40,8 @@ ext {
     scalaMajDotMin = "$scalaMajor.$scalaMinor"
 }
 
+task('buildAndTestAll', description: 'Builds all the sub-projects as well as runs there tests.', group: 'release')
```

Comment:
I'm not super happy with this name, but I haven't thought of anything better yet.